etc/GenericCamera-101/primary_hdu.header

```diff
@@ -0,0 +1,75 @@
+SIMPLE  =                    T /
+BITPIX  =                    8 / bits per data value
+NAXIS   =                    0 / number of axes
+EXTEND  =                    T / Extensions are permitted
+ORIGIN  = 'LSST DM Header Service'/ FITS file originator
+
+        ---- Date, night and basic image information ----
+TIMESYS = 'TAI'                / The time scale used
+DATE    =                      / Creation Date and Time of File
+DATE-OBS=                      / Date of observation (image acquisition)
+DATE-BEG=                      / Time at the start of integration
+DATE-END=                      / end date of the observation
+MJD     =                      / Modified Julian Date that the file was written
+MJD-OBS =                      / Modified Julian Date of observation
+MJD-BEG =                      / Modified Julian Date derived from DATE-BEG
+MJD-END =                      / Modified Julian Date derived from DATE-END
+OBSID   =                      / ImageName from Camera StartIntergration
+BUNIT   = 'adu     '           / Brightness units for pixel array
+
+        ---- Telescope info, location, observer ----
+FACILITY= 'Vera C. Rubin Observatory' / Facility name
+TELESCOP=                      / Telescope name
```

Comment:
The Confluence page has been updated to fill in this keyword.

Reply:
Fixed too. Thanks for noticing.